examples/.explainer-config.toml

```diff
@@ -0,0 +1,9 @@
+# Any of these config options can or can not be provided
+# Feel free to place this file in your home directory, i.e: cp examples/.explainer-config.toml ~
+
+apiKey = "YOUR_API_KEY"
+baseURL = "https://api.groq.com/"
+temperature = 0.8 # between 0.1 and 2

```

Comment:
this should follow the already defined default value for temperature in `index.js`
https://github.com/aamfahim/explainer.js/blob/29c31bd8ab656243296f7370bf41014c40ca75fe/index.js#L23